app/views/editor/level/tasks/TasksTabView.coffee

```diff
@@ -0,0 +1,149 @@
+CocoView = require 'views/core/CocoView'
+template = require 'templates/editor/level/tasks-tab'
+Level = require 'models/Level'
+
+module.exports = class TasksTabView extends CocoView
+  id: 'editor-level-tasks-tab-view'
+  className: 'tab-pane'
+  template: template
+  events:
+    'click .taskRow': 'onClickTaskRow'
+    'click .taskInput': 'onClickTaskInput'
+    'click .startEdit': 'onClickStartEdit'
+    'click #createTask': 'onClickCreateTask'
+    'keydown #curEdit': 'onKeyDownCurEdit'
+
+  subscriptions:
+    'editor:level-loaded': 'onLevelLoaded'
+
+  defaultTaskLinks:
+    # Order doesn't matter.
+    'Name the level.':'./'
+    'Create a Referee stub, if needed.':'./'
+    'Build the level.':'./'
+    'Set up goals.':'./'
+    'Choose the Existence System lifespan and frame rate.':'./'
+    'Choose the UI System paths and coordinate hover if needed.':'./'
+    'Choose the AI System pathfinding and Vision System line of sight.':'./'
+    'Write the sample code.':'./'
+    'Do basic set decoration.':'./'
+    'Adjust script camera bounds.':'./'
+    'Choose music file in Introduction script.':'./'
+    'Choose autoplay in Introduction script.':'./'
+    'Add to a campaign.':'./'
+    'Publish.':'./'
+    'Choose level options like required/restricted gear.':'./'
+    'Create achievements, including unlocking next level.':'./'
+    'Choose leaderboard score types.':'./'
+    'Playtest with a slow/tough hero.':'./'
+    'Playtest with a fast/weak hero.':'./'
+    'Playtest with a couple random seeds.':'./'
+    'Make sure the level ends promptly on success and failure.':'./'
+    'Remove/simplify unnecessary doodad collision.':'./'
+    'Release to adventurers via MailChimp.':'./'
+    'Write the description.':'./'
+    'Add i18n field for the sample code comments.':'./'
+    'Add Clojure/Lua/CoffeeScript.':'./'
+    'Write the guide.':'./'
+    'Write a loading tip, if needed.':'./'
+    'Click the Populate i18n button.':'./'
+    'Add programming concepts covered.':'./'
+    'Mark whether it requires a subscription.':'./'
+    'Release to everyone via MailChimp.':'./'
+    'Check completion/engagement/problem analytics.':'./'
+    'Do thorough set decoration.':'./'
+    'Add a walkthrough video.':'./'
+
+  taskMap: ->
+    return @tasks?.map((_obj) -> return (name: _obj.get('name'), complete: (_obj.get('complete') || false)))
+
+  taskArray: ->
+    return @tasks?.toArray()
+
+  findTask: (_name) ->
+    return @tasks.findWhere(name:_name)
+
+  onLoaded: ->
+  onLevelLoaded: (e) ->
+    @level = e.level
+    Task = Backbone.Model.extend({
+      initialize: ->
+        if e?.level?._revertAttributes?.tasks?
```

Comment:
There's probably no easy way to do this given the revert system. So a comment explaining what's going on here would help.